@xen-orchestra/lite/src/stores/index.ts

```diff
@@ -22,8 +23,14 @@ export function createRecordContext<T extends XenApiRecord>(
     isInitialized = true;
 
     const xapiRecordsStore = useRecordsStore();
-    await xapiRecordsStore.loadRecords(objectType);
-    isReady.value = true;
+    try {
+      await xapiRecordsStore.loadRecords(objectType);
+    } catch (error) {
+      hasError.value = true;
+      console.error("stores#createRecordContext", error);
+    }
+
+    isLoading.value = false;
```

Comment:
It's a good practice to have a `finally` block to handle UI related state.

```suggestion
    } finally {
      isLoading.value = false;
    }
```